golemcli/src/commands/acl.rs

```diff
@@ -233,6 +234,10 @@ impl Section {
                                     .as_golem_net()
                                     .block_node(identity, timeout)
                                     .from_err()
+                                    .and_then(move |( _result, exist, _message )| {
```

Comment:
maybe if result is false, you should print the error message?

Reply:
yep, would be better